apps/bfd-data-npi/src/main/java/gov/cms/bfd/data/npi/lookup/NPIOrgLookup.java

```diff
@@ -0,0 +1,126 @@
+package gov.cms.bfd.data.npi.lookup;
+
+import gov.cms.bfd.data.npi.utility.App;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Provides an NPI org lookup. */
+public class NPIOrgLookup {
+  private static final Logger LOGGER = LoggerFactory.getLogger(NPIOrgLookup.class);
+
+  /** A fake npi number used for testing. */
+  public static final String FAKE_NPI_NUMBER = "0000000000";
+
+  /** A fake org name display that is associated with the FAKE_NPI_ORG_NAME. */
+  public static final String FAKE_NPI_ORG_NAME = "Fake ORG Name";
+
+  /** Hashmap to keep the org names. */
+  private final Map<String, String> npiOrgHashMap = new HashMap<>();
+
+  /** A field to return the testing org lookup. */
+  private static NPIOrgLookup npiOrgLookupForTesting;
+
+  /** A field to return the production org lookup. */
+  private static NPIOrgLookup npiOrgLookupForProduction;
+
+  /**
+   * Factory method for creating a {@link NPIOrgLookup } for testing that includes the fake org
+   * name.
```

Comment:
The ctor seems a little wierd; why not have a single factory method that takes in the boolean (something like boolean forTestingOnly). In both cases we return NPIOrgLookup object.